deploy/osd-delete-backplane-serviceaccounts/00-delete-backplane-serviceaccounts.rbac.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: backplane
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-delete-serviceaccounts
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - delete
+---
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+    - allowFirst: true
+      clusterRoleName: osd-delete-serviceaccounts
+      namespacesAllowedRegex: "(^openshift-backplane-.*)"
+  subjectKind: User
+  subjectName: system:serviceaccount:backplane:osd-delete-backplane-serviceaccounts
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-get-namespace
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: osd-backplane-namespaces
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: osd-get-namespace
+subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: system:serviceaccount:backplane:osd-delete-backplane-serviceaccounts
```

Comment:
Not going to hold up the PR for it, but FYI you can manage global perms with SubjectPermission as well, see `.spec.clusterPermissions`.  But it does what this is doing so :+1: to what you have.